README.md

```diff
@@ -1 +1,72 @@
-# oracle-api
\ No newline at end of file
+# Oracle api.
+This api is for searching and ordering iStore catalogue items, used by rail component ordering system (rcos).
```

Comment:
While true, it is worth remembering that the library is meant to be used by anyone needing to integrate with Oracle.